oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCClientConfiguration.java

```diff
@@ -1275,15 +1287,15 @@ public AccessToken getAccessToken()
     public void setAccessToken(AccessToken accessToken, RefreshToken refreshToken)
     {
         if (isAuthenticationConfiguration()) {
-            // Don't store the BearerAccessToken object directly as it could cause classloader problems when an extension is
```

Comment:
Better avoid all this reformatting which is not related to the issue you are working on.